Dockerfile

```diff
@@ -248,6 +383,21 @@ RUN cmake -S contrib/tenzir-plugins/vast -B build-vast -G Ninja \
 
 FROM tenzir-de AS tenzir-ce
 
+COPY --from=google-cloud-pubsub-plugin --chown=tenzir:tenzir /plugin/google-cloud-pubsub /
```

Comment:
Nit: Aside from this, all the plugins are sorted alphabetically. Google Cloud Pub/Sub is also at the top above.